pmdarima/setup.py

```diff
@@ -4,9 +4,8 @@
 #
 # Setup for submodules of pmdarima
 
-from __future__ import absolute_import
-
 import os
+import sys
```

Comment:
Don't think this is used except for the `TODO`

Reply:
True 👀